price-set.go

```diff
@@ -0,0 +1,5 @@
+package httpshopify
+
+type PriceSetDTO struct {
```

Comment:
I dont think your precommit hook is working, this shouldnt have let you commit without a comment